### PR TITLE
BE-45 Implement text extraction method from image files

### PR DESCRIPTION
**Pull request by MaliniakovaKarin**

### CODE REVIEW

**Review by KToldy — DISMISSED** [review]
Looks good, no remarks

File: README.md [hunk 1]
```diff
@@ -24,4 +24,9 @@ To create all necessary tables in database, run:
 ## Code formatting
 Before committing, format code using `black` formater:
 
-```black .```
\ No newline at end of file
+```black .```
+
+## Text extraction from image
+Download and install tesseract (https://digi.bib.uni-mannheim.de/tesseract/).
```
Comment (MCFreddie777) [inline README.md:30]: ### Installation

 - *Windows*:

   Download the precompiled file (_tesseract-ocr-w64-setup-v4.1.0.20190314.exe_) and install tesseract from 
   here : [https://digi.bib.uni-mannheim.de/tesseract/](https://digi.bib.uni-mannheim.de/tesseract/).
    Don't forget to place the installation path into PATH environment variable.

 - *Linux (Ubuntu, debian)*: 
   ```
   sudo apt-get install tesseract-ocr
   ```
 - *MacOS*:
   ```
   brew install tesseract
   ```

File: antiplag/.env.example [hunk 2]
```diff
@@ -1,3 +1,4 @@
 DEBUG=True
 DJANGO_SECRET_KEY=3YSxRH/mcqSJpkk40s+0qy+uh+9o5LIBdZtwHOo6oNaLe8+Mx6
 DATABASE_URL=postgresql://postgres:postgres@localhost:5432/antiplag
+TESSERACT=C:\Program Files\Tesseract-OCR\tesseract.exe
```
Comment (MCFreddie777) [inline antiplag/.env.example:4]: I would name this variable `TESSERACT_PATH`

File: antiplag/nlp/text_preprocessing.py [hunk 3]
```diff
@@ -14,7 +14,17 @@
 import string
 from pycontractions import Contractions
 from textblob import TextBlob
+import pytesseract
+from PIL import Image
```
Comment (MCFreddie777) [inline antiplag/nlp/text_preprocessing.py:18]: I see you're importing pillow (or its slightly modified version - PIL) here, but it's not in requirements. Does it come with django by default or do we need to add it to requirements?

File: antiplag/nlp/text_preprocessing.py [hunk 4]
```diff
@@ -14,7 +14,17 @@
...
```
Comment (MCFreddie777) [inline antiplag/nlp/text_preprocessing.py:19]: Should we leave this blank line here? Is it indented?

**Review by MCFreddie777 — CHANGES_REQUESTED** [review]

**Comment by MCFreddie777** [comment]
Please don't forget to resolve conflicts.

**Review by MCFreddie777 — APPROVED** [review]

**Review by OliverKanat — APPROVED** [review]

**Pull request merged**